features/step_definitions/project_steps.rb

```diff
@@ -306,7 +306,7 @@
   expect(project).to_not be_nil
 end
 
-Then /^I should have (\d+) todo in project "([^"]*)"$/ do |todo_count, project_name|
+Then /^I should have (\d+) todo(?:s?) in project "([^"]*)"$/ do |todo_count, project_name|
```

Comment:
Notice I wrote `todos?` with a question mark -- that should make the last character, the s, optional in the match.